### PR TITLE
Transactions App 1.0

### PR DESCRIPTION
**Pull request by AKeeganDev**
This pull request is the first implementation of my Transaction app. This pull request allows users to:

-Create an account
-Log in
-Create expense categories
-Create expenses within those categories
-Sign out

A demo of these features are all found in my 5 minute demo [here](https://www.loom.com/share/9d8a5dc802e543f0834afad3b06afcf4)

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,68 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jsug9/recipe-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```
Comment (rloterh) [inline README.md:63]: - [x] It's a nice and smart thing to give appropriate credit to **[Gregoire Vella](https://www.behance.net/gregoirevella)** on Behance, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Aaron, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- App is running as intended
- Queries with Active Records properly established
- Good unit test coverage with RSpec


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AKeeganDev) [reply README.md:63]: Oh yes, I forgot this. Thank you for checking that. I'll update and resubmit. Thank you!

Comment (AKeeganDev) [reply README.md:63]: Oops. totally forgot it, thank you!

**Review by EricMbouwe — APPROVED** [review]
Hi @AKeeganDev, 

In my opinion, your project is ready for presentation! ⚡🔥 There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**